deb/openmediavault/usr/share/php/openmediavault/system/storage/storagedevice.inc

```diff
@@ -200,6 +200,32 @@ class StorageDevice extends \OMV\System\BlockDevice
 		return (0 == strcasecmp("ata", $property)) ? TRUE : FALSE;
 	}
 
+	/**
+	 * Check if the given device is an SD or MMC Device.
+	 * @return boolean true if device is SD or MMC, otherwise FALSE.
+	 */
+	public function isMmcblk() {
+		// Try to identify SD card by 'ID_PATH=xxx.mmc'
+		// Example (raspberry-pi):
+		// ID_PATH=platform-3f202000.mmc
+		if (TRUE === $this->hasUdevProperty("ID_PATH")) {
```

Comment:
Is this code block really necessary. Are there devices that are not named ´/dev/mmcxxx´?

Reply:
No, I've removed the block.